### ISSUE TITLE
e.setups is undefined with "latest" embedded component

### ISSUE THREAD

**Issue by csparker247** [Smithsonian/dpo-voyager#140]
I've just setup a site using the embedded component as suggested using the basic example (option 2) [in the docs](https://smithsonian.github.io/dpo-voyager/explorer/usage/): 
```html
<voyager-explorer root='data/' document='document.json'></voyager-explorer>

<!-- This is the required javascript file for the Voyager element. -->
<script src="https://3d-api.si.edu/resources/js/voyager-explorer.min.js"></script>
```

When I put this site on my server, Voyager loads but displays the error `Failed to load document: e.setups is undefined`.  If I copy-and-paste the `setups` section from the [default template](https://github.com/Smithsonian/dpo-voyager/blob/master/source/client/templates/default.svx.json) into my `document.json`, the error goes away. So it seems like maybe there's an issue loading the default setup?

If I change the script src to the latest tag version `https://3d-api.si.edu/resources/js/v0.17.5/voyager-explorer.min.js`, that also fixes the issue. So maybe you've fixed this bug already, but the "latest" uri isn't pointing to an updated source file?

**Comment by gjcope** [comment]
Hi @csparker247 ,
Could you attach your original document.json that causes the error? This is not a bug I've seen, so I'd like to reproduce it. It's very odd that updating to the tagged version fixed it, as those endpoints are both currently serving the same file. It did just get deployed this morning, so maybe you caught it before it propagated to "latest".

**Comment by csparker247** [comment]
Sure, you can find that [here](https://gitlab.com/educelab/dri-voyager/-/blob/project-init/data/PHerc1186Cr01.json). The live site is running on GitLab pages [here](https://educelab.gitlab.io/dri-voyager/), currently setup to use the v0.17.5 URL. 

I do currently have a CORS error on the host from my model, but that gets reportedly separately in the console, so I don't think it's related.

**Comment by gjcope** [comment]
Could you try the 'latest' link again? It may have just been a synching issue. I can confirm that a setup-related issue was resolved in v0.17.5. The error message I see with your file is slightly different, but effectively the same thing. Your GitLabs page is not currently loading for me, but that's due to the http/https mismatch as you referenced.

**Comment by gjcope** [comment]
Glad it's resolved! Let me know if you encounter any other problems and we'd also love to hear if you decide to use Voyager for any projects.
